java/test/jmri/jmrit/display/layoutEditor/LayoutEditorChecksTest.java

```diff
@@ -1,40 +1,349 @@
 package jmri.jmrit.display.layoutEditor;
 
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Assume;
-import org.junit.Before;
-import org.junit.Test;
-
-import jmri.util.JUnitUtil;
-
 import java.awt.GraphicsEnvironment;
+import java.awt.geom.*;
+import java.util.List;
+import javax.annotation.CheckForNull;
+import javax.swing.*;
+import jmri.InstanceManager;
+import jmri.jmrit.display.EditorFrameOperator;
+import jmri.util.*;
+import jmri.util.junit.rules.RetryRule;
+import org.junit.*;
+import org.junit.rules.Timeout;
+import org.netbeans.jemmy.QueueTool;
+import org.netbeans.jemmy.operators.*;
 
 /**
- *
  * @author Paul Bender Copyright (C) 2017
+ * @author George Warner Copyright: (C) 2019
  */
 public class LayoutEditorChecksTest {
 
+    @Rule   // 10 second timeout for methods in this test class.
+    public Timeout globalTimeout = Timeout.seconds(10);
+
+    @Rule   // allow 5 retries
+    public RetryRule retryRule = new RetryRule(5);
```

Comment:
I think 5 is way too high.  If a test needs 5 retries it is really too flakey to be reliable.  I really don’t want to see this higher than 3.

Reply:
Agreed… fixed.